### PR TITLE
minor textual change

### PR DESCRIPTION
**Pull request by seneca**
I'm not sure whether the author wanted to say that Nue is in a completely different ballpark _than_ what you see with Rust, etc., or whether Nue is in the ballpark _of_ Rust, etc., which would mean in the same ballpark (my change here).

### CODE REVIEW

File: packages/nuejs.org/docs/tutorial.md [hunk 1]
```diff
@@ -348,10 +348,6 @@ The production-optimized version is generated with the `nue build --production`
   large: /img/blog-generate-big.png
   width: 500
 
-This step is extremely fast. No matter how large your website, the generation times are in the ballpark than what you see with **Rust** and **Go** -based generators like **Hugo**.
+This step is extremely fast. No matter how large your website, the generation times are in the ballpark of what you see with **Rust** and **Go** -based generators like **Hugo**.
```
Comment (nobkd) [inline packages/nuejs.org/docs/tutorial.md]:
```suggestion
This step is extremely fast. No matter how large your website, the generation times are in the ballpark of what you see with **Rust** and **Go**-based generators like **Hugo**.
```

**Review by nobkd — CHANGES_REQUESTED** [review]

**Review by nobkd — APPROVED** [review]

**Comment by nobkd** [comment]
Your change sounds correct to me, but I'll let `tipiirai` look over it too.

File: packages/nuejs.org/docs/content.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ Again, the name "stack" must be implemented in your website CSS by the UX develo
 
 
 ### Grids
-[Grid](tags.html#grids) is a built-in tag used like the stack but is meant for more complex layouts. They have more configuration options and their visual appearance and behavior can be enhanced with JavaScript.
+[Grid](tags.html#grids) is a built-in tag used like the stack but is meant for more complex layouts. They have more configuration options for their visual appearance and behavior can be enhanced with JavaScript.
```
Comment (nobkd) [inline packages/nuejs.org/docs/content.md:122]: I don't think this sentence is understandable (and the change to `for` changes the meaning and breaks grammar (see `and behavior` missing `their` in between))

**Review by nobkd — CHANGES_REQUESTED** [review]

File: packages/nuejs.org/docs/content.md (hunk 2 above)
Comment (seneca) [reply packages/nuejs.org/docs/content.md:122]: I hope the author will be able to look into this.

Comment (tipiirai) [reply packages/nuejs.org/docs/content.md:122]: Makes sense. Merging this and pushing live

**Pull request merged**